### PR TITLE
Update search provider

### PR DESCRIPTION
**Pull request by fushinari**
I'm unable to test this as of yet.

### CODE REVIEW

File: dialect/search_provider/search_provider.in [hunk 1]
```diff
@@ -50,12 +53,12 @@ class TranslateService(dbus.service.Object):
...
         self.live_enabled = self.is_live_enabled()
+        text = ' '.join(terms)
         if self.live_enabled:
-            text = ' '.join(terms)
             self.translation(text)
-            return [text]
-
-        return None
+        else:
+            self.translations[text] = _('Please enable Live Translation in Dialact\'s Preferences')
```
Comment (rafaelmardojai) [inline dialect/search_provider/search_provider.in:60]: Shouldn't this be `Translate “{text}” with {active_translator}`?

That's why I meant by mimicking GNOME Web.

Comment (fushinari) [reply dialect/search_provider/search_provider.in:60]: Ah I see, will change that.

Have you been able to test this yet?

Comment (fushinari) [reply dialect/search_provider/search_provider.in]: Fixed

**Comment by fushinari** [comment]
So... I still havent been able to test this due to flatpak not building as intended. I tried renaming "SearchProvider" into "search-provider" like it seems to be for some apps. But doesn't work yet. Should we report this to flatpak instead?

**Comment by fushinari** [comment]
And sorry for the delays, life has been busy :sweat_smile:

**Comment by fushinari** [comment]
We have the approval for app.drey.Dialect .  So, if it's okay with you, we can merge and then test our theory.

**Review by rafaelmardojai — APPROVED** [review]
Code looks good, and theoretically should work fine. So lets merge and see how it plays after the app id change.

**Pull request merged**